test/dlc_tests/ec2/test_smdebug.py

```diff
@@ -18,6 +18,8 @@
 @pytest.mark.parametrize("ec2_instance_type", SMDEBUG_EC2_GPU_INSTANCE_TYPE, indirect=True)
 def test_smdebug_gpu(training, ec2_connection, region, gpu_only, py3_only):
     # TODO: Remove this once test timeout has been debugged (failures especially on p2.8xlarge)
+    if is_tf2(training) and "2.3.0" in training and SMDEBUG_EC2_GPU_INSTANCE_TYPE == "p2.8xlarge":
```

Comment:
The condition for `is_tf2(image_uri)` is just a more generic version of `"2.3.0" in image_uri`.

Can make do with just the "2.3.0" check.

Reply:
In more detail -- `is_tf2(image_uri)` checks if the image version matches the format "2.x.y".

https://github.com/aws/deep-learning-containers/blob/e548c7566eb5b5b9d8aeeff8a28f275f66178996/test/test_utils/__init__.py#L66-L69

Reply:
it also adds the "tensorflow" condition though which is important. I don't think it hurts here.

Reply:
> it also adds the "tensorflow" condition though which is important. I don't think it hurts here.

Alright. I'll edit my comment above. I should have put in "nit:" ahead of my comment.